src/context/fetch.ts

```diff
@@ -26,6 +26,21 @@ export const augmentRequestInit = (requestInit: RequestInit): RequestInit => {
   }
 }
 
+const createFetchRequestParameters = (input: MockedRequest) => {
+  const { body } = input
+  const requestParameters: RequestInit = {
+    ...input,
+    body: null,
+  }
+
+  if (input.method !== 'GET' && input.method !== 'HEAD') {
```

Comment:
Nitpick: I suggest you also get the `method` property from the `input` destructuring at the top of this function, the same as you do with the `body` property.